src/main/java/de/retest/web/selenium/RecheckDriver.java

```diff
@@ -0,0 +1,29 @@
+package de.retest.web.selenium;
+
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.firefox.FirefoxDriver;
+import org.openqa.selenium.remote.RemoteWebDriver;
+
+import de.retest.recheck.Recheck;
+import de.retest.recheck.RecheckOptions;
+
+/**
+ * This is a wrapper for different {@link RemoteWebDriver} implementations, such as {@link ChromeDriver} or
+ * {@link FirefoxDriver}.
+ *
+ * The RecheckDriver extends AutocheckingDriver, such that it is unnecessary to call
```

Comment:
Would you mind wrapping both classes in `{@code }` such that they get rendered with a monospaced font?